evm/src/main/java/org/hyperledger/besu/evm/MainnetEVMs.java

```diff
@@ -816,7 +1010,7 @@ public static void registerCancunOperations(
    * @return the evm
    */
   public static EVM futureEips(final BigInteger chainId, final EvmConfiguration evmConfiguration) {
-    return shanghai(chainId, evmConfiguration);
+    return futureEips(new CancunGasCalculator(), chainId, evmConfiguration);
```

Comment:
Can't comment inline as it's not in the diff, but I think the experimentalEips equivalent method is missing `new CancunGasCalculator()`

Reply:
Why do we need the new it up at this level anyway?

Reply:
Every EVM needs a gas calculator.  I suppose we could use static instance since they don't have state.

Reply:
Making it configurable also allows for gas schedules to be change by downstream consumers.